src/D2L.CodeStyle.Analyzers/Helpers/TypeAllowList.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Text;
+
+namespace D2L.CodeStyle.Analyzers.Helpers {
+
+	internal sealed class TypeAllowedList {

```

Comment:
Yep, you're probably right. I'll change this.

Reply:
There's another similar type list that might need the same treatment.

Reply:
Ok, it's a directory called `TestAllowedLists`. It's a bit clunky, but it's more accurate than `AllowedTestLists`, which sounds nicer but unfortunately doesn't carry the same meaning. I'm inclined to leave it as is, but if you have a preference otherwise, I can change it.